build/mvn

```diff
@@ -36,7 +36,7 @@ _DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Preserve the calling directory
 _CALLING_DIR="$(pwd)"
 # Options used during compilation
-_COMPILE_JVM_OPTS="-Xss128m -Xmx4g -XX:ReservedCodeCacheSize=128m"
+_COMPILE_JVM_OPTS="-Xss128m -Xmx4g -XX:ReservedCodeCacheSize=128m -Dmaven.resolver.transport=wagon"
```

Comment:
try to specify `-Dmaven.resolver.transport=wagon`

Reply:
Hi, a kind ping from Kyuubi, we reverted to using Maven 3.8.x in Kyuubi for CI builds and local usage, to reduce side effects and easier to use.

Reply:
Thanks for your sync @bowenliang123